### PR TITLE
[minor] Support AWS MSK as Kafka provider

### PR DESCRIPTION
**Pull request by padmankosalaram**
- [x] add support for aws msk provision and deprovision
- [x] aws provision was tested as part of https://github.ibm.com/maximoappsuite/tracker-hyperscalers/issues/239
- [x] aws deprovision was tested as part of https://github.ibm.com/maximoappsuite/tracker-hyperscalers/issues/240 

Please review the pr and provide your comments.

### CODE REVIEW

**Review by durera — CHANGES_REQUESTED** [review]
Also need changelog updates as this is a new feature

File: ibm/mas_devops/roles/kafka/README.md [hunk 1]
```diff
@@ -186,7 +186,183 @@ Example Playbook
...
 ```
 
+AWS MSK Role Variables
+-------------------------------------
+
+### kafka_action
+Action to be performed by Kafka role. Valid values are `provision` or `deprovision`. To provision AWS MSK kafka cluster, set this variable as `provision`. To deprovision an existing AWS MSK kafka cluster, set this variable as `deprovision`. 
+
+- Environment Variable: `KAFKA_ACTION`
+- Default Value: `provision`
```
[the remark this answers is not in the document]
Comment (padmankosalaram) [reply ibm/mas_devops/roles/kafka/README.md]: KAFKA_ACTION provision / deprovision was updated as install / uninstall. 
The related file names were modified to install.yml and uninstall.yml + reference in ReadMe

[the remark this answers is not in the document]
Comment (padmankosalaram) [reply ibm/mas_devops/roles/kafka/README.md]: Change logs was updated with PR 727 details

**Comment by padmankosalaram** [comment]
Tested as part of https://github.ibm.com/maximoappsuite/tracker-hyperscalers/issues/265

**Review by whitfiea — DISMISSED** [review]

**Comment by padmankosalaram** [comment]
@durera @whitfiea @andrercm - This PR is ready for merge. I done with the review comments and testing. Please help with merge

**Review by whitfiea — APPROVED** [review]

**Pull request merged**